alacritty/src/window.rs

```diff
@@ -113,10 +114,16 @@ fn create_gl_window<E>(
         window = window.with_inner_size(dimensions);
     }
 
+    let hardware_acceleration = match window_config.energy {
+        Energy::Performance => Some(true),
+        Energy::Powersave => Some(false),
```

Comment:
Have you verified that on macOS it'll use integrated GPU even on AC? The same question for `Some(true)`, will it use discrete GPU even on battery?

Reply:
That's a great question. I completely forgot to mention that this PR is not sufficiently reviewed yet when it comes to the feature itself. While #3587 states that `with_hardware_acceleration(Some(true))` enables the dedicated GPU for people, on my system the dedicated GPU is never used for Alacritty according to the system manager.

So I'll have to look into why that is, since other applications like iTerm seem to use it just fine.

However that won't happen until I have my macbook around for a longer time on monday.